task/fbc-fips-check-oci-ta/0.1/fbc-fips-check-oci-ta.yaml

```diff
@@ -190,7 +190,14 @@ spec:
           cpu: 100m
           memory: 256Mi
       ref:
-        name: fips-operator-check-step-action
+        params:
+          - name: url
+            value: https://github.com/konflux-ci/build-definitions
+          - name: revision
+            value: c366291f484f2510767a51d0f45d7bc37877e94e
```

Comment:
What is the plan for keeping this reference up to date? Will it be automated? The commit is already not the latest one (not that it makes sense for it to be the latest one since this stepaction hasn't changed).

Should we instead take a generic approach similar to how we inject the task bundles to pipelines? https://github.com/konflux-ci/build-definitions/blob/main/hack/build-and-push.sh#L193-L211

Reply:
For now, the revision would have to be updated manually whenever we change the stepaction.

I think it does make sense to automate the process as we have an increasing number of stepactions. May I suggest tracking it in a separate Jira?

Reply:
filed KONFLUX-6629